lib/fabrik/VirtualHostInstance.js

```diff
@@ -0,0 +1,93 @@
+const _ = require('lodash');
+const Promise = require('bluebird');
+const BaseInstance = require('./BaseInstance');
+const bosh = require('../bosh');
+const utils = require('../utils');
+const mapper = require('./VirtualHostRelationMapper');
+const catalog = require('../models').catalog;
+
+class VirtualHostInstance extends BaseInstance {
+
+  constructor(guid, manager) {
+    super(guid, manager);
+    this.director = bosh.director;
+    this.mapper = mapper.VirtualHostRelationMapper;
+  }
+
+  get async() {
+    return false;
+  }
+
+  initialize() {
+    return this.getDeploymentNameForVirtualHostInstanceId(this.guid)
+      .then((deploymentName) => this.deploymentName = deploymentName);
```

Comment:
can remove paranthesis around `deploymentName` as a convention when there is single param, its avoided in our code base.